cmd/seccomp-operator/main.go

```diff
@@ -75,6 +75,19 @@ func main() {
 				return nil
 			},
 		},
+		&cli.Command{
+			Name:    "initialize",
+			Aliases: []string{"init", "i"},
+			Usage:   "initialize the operator directories and set the right permissions",
+			Action: func(c *cli.Context) error {
+				return errors.Wrap(initialize.SetupRootless(
+					config.KubeletSeccompRootPath,
```

Comment:
Might be nice to be able to override this if desired. I'm fine with deferring that until later though 👍

Reply:
Yeah, I can follow-up on that. We could add a `Config` structure to the `config` package and let it be populated by CLI flags and/or environment variables. Then we could pass them there…